canonical/jaas-dashboard
# Use relative time in action log

## Done

- Change time in the action log to be relative and display the timestamp in a tooltip.
- Update tests to TypeScript/RTL.

## QA

- Go to a model that has action logs (you might need to perform an action for an app).
- Check that the completion time column shows relative dates.
- Hover the date and you should see the exact date/time.

## Details

Fixes: #965.

## Screenshots


<img width="1016" alt="Screen Shot 2022-10-24 at 12 19 45 pm" src="https://user-images.githubusercontent.com/361637/197439207-aff44f96-0b6e-4fac-8739-4e72d4cf32cc.png">
